examples/flutter_gallery/test/drawer_test.dart

```diff
@@ -54,16 +54,34 @@ void main() {
     // Switch back to system theme setting: first menu button, choose 'System Default'
     await tester.tap(find.byIcon(Icons.arrow_drop_down).first);
     await tester.pumpAndSettle();
-    await tester.tap(find.text('System Default').at(1));
+    await tester.tap(find.text('System Default').at(2));
```

Comment:
Is there a way to particularly target this instance of `Text('System Default')` here and elsewhere? I was thinking something like: 
```
  find.descendant(
    of: find.byType(PopupMenu),
    matching: find.text('System Default')
  )
```
However, it seems both _PopupMenu and the _VisualDensityItem widgets are private, so I'm not sure if it's possible

Reply:
That's pretty neat, I'll definitely keep this in mind when I need it in the future!